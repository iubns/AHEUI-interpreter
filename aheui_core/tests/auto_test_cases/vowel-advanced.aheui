반븓븝불
우멍벎망이밟망희
빈
빋밟망희
붑으
발몽
ㅇ밞망분
ㅇ불법벋
의멍밞망희

ㅡ - 커서가 가로로 왔으면 그 방향대로, 세로로 왔으면 전에 있던 자리로 옮깁니다.
ㅣ - 커서가 세로로 왔으면 그 방향대로, 가로로 왔으면 전에 있던 자리로 옮깁니다.
ㅢ - 커서를 전에 있던 자리로 옮깁니다.