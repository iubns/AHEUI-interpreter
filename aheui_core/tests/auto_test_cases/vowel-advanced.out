543295432954329